ext/fileinfo/libmagic/apprentice.c

```diff
@@ -185,7 +185,11 @@ private struct {
 	{ NULL, 0, NULL }
 };
 
+#if defined(__GNUC__) || defined(__clang__)
+#include "../data_file_string.c"
+#else
 #include "../data_file.c"
+#endif
```

Comment:
I hesitated inverting the condition (`#ifndef PHP_WIN32` or alike),
but I prefer being careful not to break anything,
and as such whitelist clearly compatible compilers one by one in the #if.